src/components/MovieList/MovieList.jsx

```diff
@@ -1,5 +1,16 @@
+import { MovieCard } from '../MovieCard';
 import './MovieList.scss';
 
-export const MovieList = () => (
-  <>Put the list here</>
-);
+export const MovieList = ({ movies }) => {
+  if (movies.length === 0) {
+    return null;
+  }
```

Comment:
not the best idea to return null, preferably to return fallback text, but you might have problems with tests because of it

Reply:
Yes, usually if you don't have something to render you just don't render the component at all, and render another fallback instead.